.travis.yml

```diff
@@ -11,27 +11,33 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 7.2
   - nightly
 
 matrix:
   include:
     - php: 5.5
       env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
-      env: SYMFONY_VERSION="2.8.*@dev"
-    - php: 5.6
+      env: SYMFONY_VERSION="2.8.*"
+    - php: 7.0
+      env: SYMFONY_VERSION="3.2.*"
```

Comment:
don't had it. 3.2 is unmaintained

Reply:
Not exactly true, end of support is in January. Until then we need to make sure it works. If you notice, there are BC layers introduced in my patch and I would like them to get tested.